kolibri/plugins/learn/assets/src/views/content-page.vue

```diff
@@ -9,22 +9,23 @@
       :value="content.num_coach_contents"
     />
 
-    <content-renderer
-      v-if="!content.assessment"
-      class="content-renderer"
-      @sessionInitialized="setWasIncomplete"
-      @startTracking="startTracking"
-      @stopTracking="stopTracking"
-      @updateProgress="updateProgress"
-      :id="content.id"
-      :kind="content.kind"
-      :files="content.files"
-      :contentId="content.content_id"
-      :channelId="channelId"
-      :available="content.available"
-      :extraFields="content.extra_fields"
-      :initSession="initSession"
-    >
+    <template v-if="!content.assessment">
```

Comment:
I removed the slot from the content renderer because It makes sizing the content-renderer harder. For example setting a max-height on the content-renderer with scroll set to overflow will cause the button to also scroll. I think the button should be separate from the content-renderer.

Reply:
Since both options in the `if/else` use a button, why can't we recycle the same button across both? Wouldn't that let us `v-if` only the component?

Reply:
Yes, this can be simplified, I think.